scripts/sonarscan.sh

```diff
@@ -2,17 +2,23 @@
 
 set -e
 
-export OUTPUTDIR="coverage"
-mkdir -p "$OUTPUTDIR"
+if [ -n "$INPUT_SONAR_PROJECT_BASE_DIR" ]; then
+  echo "Changing to project dir $INPUT_SONAR_PROJECT_BASE_DIR"
+  cd "$INPUT_SONAR_PROJECT_BASE_DIR"
+fi
 
 SONAR_ORGANIZATION="$SONAR_ORG"
 
+if [ -n "$INPUT_SONAR_PROJECT_KEY" ]; then
+  echo "Changing project key to $INPUT_SONAR_PROJECT_KEY"
+  SONAR_PROJECT_KEY="$INPUT_SONAR_PROJECT_KEY"
+fi
+
 sonar_args="-Dsonar.organization=$SONAR_ORGANIZATION \
             -Dsonar.projectKey=$SONAR_PROJECT_KEY \
             -Dsonar.host.url=https://sonarcloud.io \
             -Dsonar.login=$SONAR_TOKEN \
             -Dsonar.scm.disabled=true \
-            -Dsonar.javascript.lcov.reportPaths=$OUTPUTDIR/lcov.info \
```

Comment:
@LeeHughsVariant do you think this is ok to remove? I thought teams should specify their own coverage paths instead of making an assumption.

Reply:
@jjosef-rsi I believe this may break for other nodejs repositories, however, I may be wrong. Are you working on any other nodejs repositories? If you are, please check these changes on other nodejs reposiotires.

Reply:
I'll test it on another repo

Reply:
it seemed to work fine https://github.com/variant-inc/hometime-requests-approval-handler/runs/4093979379?check_suite_focus=true

Reply:
@rchengvariant you good now?

Reply:
Looks good to me. If other teams are specifying their own paths. I'll go ahead and approve, should wait for approval from Vibin as well.

Reply:
I didn't even specify the path it automatically picks it up if you don't